src/styles/blocks/card.css

```diff
@@ -0,0 +1,14 @@
+body {
+  margin: 0;
+  font-size: 12px;
+  background-color: rgba(255, 255, 255, 1);
+}
+
```

Comment:
all general stylling should be done in seperate css file

Reply:
done